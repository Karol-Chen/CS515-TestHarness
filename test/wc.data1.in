test
meow meow meow

1111