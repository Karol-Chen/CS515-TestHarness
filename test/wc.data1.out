4	5	25	../test/wc.data1.in
